src/redisraft.c

```diff
@@ -1874,6 +1883,18 @@ static int registerRaftCommands(RedisModuleCtx *ctx)
     return REDISMODULE_OK;
 }
 
+void moduleChangeCallback(RedisModuleCtx *ctx, RedisModuleEvent e, uint64_t sub, void *data)
+{
+    REDISMODULE_NOT_USED(e);
+
+    if (sub != REDISMODULE_SUBEVENT_MODULE_LOADED) {
```

Comment:
If we are doing that, I guess we should also do it for `REDISMODULE_SUBEVENT_MODULE_UNLOADED` as well to clear table.

I didn't try these btw, hope it works :)